Packs/Pcysys/Scripts/PenteraDynamicTable/PenteraDynamicTable.py

```diff
@@ -0,0 +1,41 @@
+import demistomock as demisto
+from CommonServerPython import *
+
+
+def pentera_dynamic_table(incident: dict = None):
+    if not incident:
+        raise ValueError('No incident')
+
+    custom_fields = incident.get('CustomFields')
+    if not custom_fields:
+        raise ValueError('No custom fields')
+
+    operation_type = custom_fields.get('penteraoperationtype')
+    if not operation_type:
+        raise ValueError('No Pentera Operation Type')
+
+    details_grid = custom_fields.get('penteraoperationdetails')
+    if not details_grid:
+        raise ValueError('No details grid')
+
+    details_table = tableToMarkdown(operation_type, details_grid)
+    return details_table
+
+
+def main():
+    try:
+        incident = demisto.incidents()[0]
+        details_table = pentera_dynamic_table(incident)
+        results = {
+            'ContentsFormat': formats['markdown'],
+            'Type': entryTypes['note'],
+            'Contents': details_table
+        }
+        demisto.results(results)
```

Comment:
use `return_outputs` instead

Reply:
I need a markdown format in 'ContentsFormat'. How can this be done with the`return_outputs`?

Reply:
just pass only the `readable_output` arg to it and it will handle it

Reply:
e.g.: `return_output(readable_output=details_table)`?

Reply:
yes